lib/livebook/runtime/standalone_init.ex

```diff
@@ -105,7 +105,7 @@ defmodule Livebook.Runtime.StandaloneInit do
         {:DOWN, ^port_ref, :port, _object, _reason} ->
           {:error, "Elixir process terminated unexpectedly"}
       after
-        10_000 ->
+        100_000 ->
```

Comment:
How much time did it take for your app? Would 30s/60s work or is it that long?

Reply:
Good question let me experiment and get back!

Reply:
Looks like it takes about 40 seconds with fresh deps to get, so 60secs could work

Reply:
We run `mix deps.get` and `mix compile` beforehand, so this timeout is only for the node startup. The timeout is from starting the child node and waiting from a message from there, so it's actually interesting what takes so much time, @josevalim any ideas?

Reply:
ah that makes sense actually I tried to replicate the issue using an app with a deliberately slow compile time and couldn't replicate. Let me time the start up time. 

(BTW The app in use is actually from a company that Dashbit consult on so we could look through that channel too if you wanted to see the specific app).

Reply:
Hey @Adzz, any conclusions on the start up time?

Reply:
@jonatanklosko Aplogies! I think what was happening is we were attempting to and failing to connect to an external service on start up so it was attempting to do that and timing out, leading to a longer time. However, our startup time is still quite slow, so a bump in the timeout would help.

Would you be happy with 40secs? Not clear on the downside of a longer timeout!

Reply:
@Adzz no worries! 40s sounds good, please also add a comment like "Use a longer timeout to account for longer child node startup, as may happen when starting with Mix"

Reply:
done